### PR TITLE
nixos/bind: formatted with nixpkgs-fmt

### PR DESCRIPTION
**Pull request by paumr**
###### Motivation for this change
Moved changes after formatting (with `nixpkgs-fmt`) to separate branch (as requested in #87973)

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: nixos/modules/services/networking/bind.nix [hunk 1]
```diff
@@ -125,14 +127,16 @@ in
...
         ";
-        example = [{
-          name = "example.com";
-          master = false;
-          file = "/var/dns/example.com";
-          masters = ["192.168.0.1"];
-          slaves = [];
-          extraConfig = "";
-        }];
+        example = [
```
Comment (Ma27) [inline nixos/modules/services/networking/bind.nix:130]: while you're at it - please use `literalExample` here for better formatting in the manual.

File: nixos/modules/services/networking/bind.nix [hunk 2]
```diff
@@ -33,29 +33,31 @@ let
       ${cfg.extraConfig}
 
       ${ concatMapStrings
-          ({ name, file, master ? true, slaves ? [], masters ? [], extraConfig ? "" }:
+      (
+        { name, file, master ? true, slaves ? [], masters ? [], extraConfig ? "" }:
```
Comment (Ma27) [inline nixos/modules/services/networking/bind.nix:37]: This formatting seems odd to me. Can anyone of the `nixpkgs-fmt` maintainers explain to me whythis is better than the original version?

**Comment by paumr** [comment]
updated initial PR to use `nixpkgs-fmt version 0.9.0`

File: nixos/modules/services/networking/bind.nix (hunk 1 above)
Comment (paumr) [reply nixos/modules/services/networking/bind.nix]: fixed

File: nixos/modules/services/networking/bind.nix (hunk 2 above)
Comment (paumr) [reply nixos/modules/services/networking/bind.nix]: might be related to this?
https://github.com/nix-community/nixpkgs-fmt/issues/126

File: nixos/modules/services/networking/bind.nix [hunk 3]
```diff
@@ -103,36 +103,40 @@ in
...
         ";
-        example = [{
-          name = "example.com";
-          master = false;
-          file = "/var/dns/example.com";
-          masters = ["192.168.0.1"];
-          slaves = [];
-          extraConfig = "";
-        }];
+        example = literalExample ''
```
Comment (alyssais) [inline nixos/modules/services/networking/bind.nix:128]: Why this change?

Comment (paumr) [reply nixos/modules/services/networking/bind.nix:128]: @Ma27 requested it:
https://github.com/NixOS/nixpkgs/pull/95050#discussion_r467759793

Comment (Ma27) [reply nixos/modules/services/networking/bind.nix:128]: @alyssais `literalExample` results in better formatting in the manual.

**Review by raboof — APPROVED** [review]

File: nixos/modules/services/networking/bind.nix (hunk 2 above)
Comment (paumr) [reply nixos/modules/services/networking/bind.nix]: This was fixed with a newer version of nixpkgs-fmt

**Comment by raboof** [comment]
I'm afraid this is now conflicting with master. Do you want me to give rebasing a try or would you prefer to pick this up yourself?

**Comment by fricklerhandwerk** [comment]
LGTM.
